### DISCUSSION TITLE
Experiencing frequent crashes on Regolith 3.2 and Ubuntu 24

### DISCUSSION THREAD

**Discussion by bkjohnson** [regolith-linux/regolith-desktop#1069] [Q&A]
A few weeks ago I installed Ubuntu 24 as a clean install and then installed Regolith 3.2. Since then I've been getting frequent crashes, often 2-3 times a day. I haven't noticed a specific pattern to them other than it seems to happen as I'm doing something, such as typing, and when it happens the only option I have is to log out. I'm not really sure what log files would be useful but I'm happy to provide some. Right now I don't think I can predict when I'll get a crash, so I'm not sure where to begin troubleshooting.

**Comment by bkjohnson** [comment]
Here is what I see when it happens - it covers the full desktop:
![crash](https://github.com/user-attachments/assets/8ec16bb0-cfc7-4251-b1f2-df33242a0ce9)

**Comment by epsilon9125** [comment]
Same here. Any updates?

**Comment by abacao** [comment]
Same... I'm having some messages in my dmesg.

It seems to happen when I'm away from the laptop and session tries to lock itself

```
gnome-flashback[3959]: segfault at 50 ip 00007aa758358203 sp 00007fff816357d0 error 4 in libgtk-3.so.0.2409.32[7aa758083000+3ae000] likely on CPU 9 (core 5, socket 0)
...
gnome-flashback[65370]: segfault at 5f0e0007be4c ip 00007b343ffa08fb sp 00007fff9ef85780 error 4 in libX11.so.6.4.0[7b343ff82000+90000] likely on CPU 2 (core 12, socket 0)
```

**Comment by bkjohnson** [comment]
For me the trigger seems to be shutting down a rails server. I can `Ctrl` + `c`, let it gracefully shut down, but then whatever I do next will result in the crash. Even just switching to another workspace.

**Comment by kgilmer** [comment]
Sorry don't have any fixes to provide unfortunately.  Often these kinds of issues can be associated with some graphics hardware/driver combo.  FWIW I have found Regolith to be very stable using intel and amd integrated graphics, as well as discrete Radeon 6xxx and 7xxx cards.

Of course, your issue may not have anything to do with that, just a guess.  Hopefully a solution emerges..

**Comment by abacao** [comment]
I'm having that issue and I have documented somethings in here: https://bugs.launchpad.net/ubuntu/+source/gnome-flashback/+bug/2086706

It happens often when the laptop is idling. 

Current work around is having "caffeeine" running to prevent laptop to logout/sleep/something that triggers the crash

**Comment by firun** [comment]
I have the same issue, but Regolith 3.2 crashes on my laptop when I open GPU intensive applications, like media players or 3d viewers. 
Any ideas how to track the issue with Regolith/gnome-flashback crashes?
